recipe/meta.yaml

```diff
@@ -92,7 +92,7 @@ requirements:
   run_constrained:
     - tomophantom=2.0.0
     - astra-toolbox>=1.9.9.dev5,<2.1
-    - tigre=2.4
+    - tigre=2.6
```

Comment:
I don't think there have been any interface changes so we could accept the range here